examples/cluster/plot_cluster_comparison.py

```diff
@@ -3,24 +3,23 @@
 Comparing different clustering algorithms on toy datasets
 =========================================================
 
-This example shows characteristics of different
-clustering algorithms on datasets that are "interesting"
-but still in 2D. With the exception of the last dataset,
-the parameters of each of these dataset-algorithm pairs
-has been tuned to produce good clustering results. Some
-algorithms are more sensitive to parameter values than
-others.
-
-The last dataset is an example of a 'null' situation for
-clustering: the data is homogeneous, and there is no good
-clustering. For this example, the null dataset uses the
-same parameters as the dataset in the row above it, which
-represents a mismatch in the parameter values and the
-data structure.
-
-While these examples give some intuition about the
-algorithms, this intuition might not apply to very high
-dimensional data.
+The following example shows the characteristic of
+different clustering algorithms applied to different two
+dimensional datasets.
+
+With the exception of the last dataset, the data are
+parametrically generated in such a way that
+the results of the respective clusterings turn out to be
+clearly distinguishable for the human reader. The aim is
+to give an intuition about how the different algorithms
+work. (It should be noted, however, that these impression
+might quickly turn invalid on higher dimensional data).
+
+The last dataset on the other hand is just rectangular
+noise, which can hold as the absence of any objectively
+useful inherent cluster in the data. Thus, any clustering
+shown is a method artefact by the respective cluster
+algorithm.
 
 """
 
```

Comment:
When we start to have cells, we like to move the imports at the beginning of the cell of the function/class will be used for the first time.

Reply:
It also good to have a marker `# %%` directly at the start because it allows execution in the IDE without the need to add a marker. This should be solved when moving the imports.